ironic/resource_ironic_node_v1.go

```diff
@@ -371,11 +435,12 @@ func resourceNodeV1Update(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	// Clean node
```

Comment:
I am not sure what to do about this.. In bmo we look at hardware info to check for raid cleaning.. not sure if this is needed, maybe just remove this?

Reply:
If it's never worked, we could just remove it unless we can do the right thing based on the hardware info? It probably makes sense that we should actually clean these nodes

Reply:
so bmo needs the bmc drivers to set the raid config, I am not sure we can do that?
https://github.com/metal3-io/baremetal-operator/blob/master/pkg/provisioner/ironic/ironic.go#L1249-L1292
Also it probably should be done in a separate PR, I'll make it obvious and we can think if we need to raise an issue for it?